### PR TITLE
fixes/m2-resubmission

### LINKED ISSUE #37: [TODO] M2 Resubmission fixes
# TODO

## 🎯 &nbsp; What is the problem that needs to be solved?

Try to be as specific as possible. Think about falsifiable outcomes.

**Refer:** [SMART Goal Examples](https://www.thebalancesmb.com/smart-goal-examples-2951827)

### 🗒 &nbsp; Summary
A brief overview of the problem to be solved. Include all relevant data points.

### ✅ &nbsp; Outcomes

- [x]  I need to fill out "bond additional funds" in order to see the results of the calculations (even if I don’t want to bond additional funds) - allow 0 as the input by default
- [x]  Update README to inform reader how to update .env file to use either our backend or host their own
    - [x]  Create the mechanism for reading variables from env file
- [x]  Implementation for info question marks as per design (dependency - design change)

## ⏳ &nbsp; ETA

TBD

If you don't specify a timezone, by default IST shall be assumed.

Refer: [ETA culture document](https://docs.google.com/document/d/1ZGa6diesOmx-jSBjm0uj8j4oN75gYqux85qA7DJOm68/edit)

### PR DESCRIPTION
**Pull request by akshatbhargava123**
closes #37 and #39 .

### CODE REVIEW

**Comment by saumyakaran** [comment]
I get the results without an input or `0` as the input but the "stake" button is un-clickable (SS Attached)

Screenshot 2020-07-23 at 11 07 07 AM

**Comment by akshatbhargava123** [comment]
@Karan, this was intentional from my side since the purpose of calculator is to:
- let users bond additional funds and not change their current nominations
- view calculations on how much they could be earnings

**Comment by saumyakaran** [comment]
The purpose of the calculator is 2-fold:

1.  To serve as an on-boarding UX for new users.
2. To allow existing users to understand how to maximize their earnings potential.

Both these purposes are addressed well in the new UX design, and there's no way to "correctly" address this in the current implementation (i.e. of the previous UX), since it didn't correctly take into account the users motivations.

I think we're on the same page that this is sub-optimal, however for right now, the user should have a way to take action even if they just want to change their nominations.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@
 	# or
 	yarn
 	```
+- Add environment variables in `.env.local`
+	```
+		NEXT_PUBLIC_API_BASE_URL=<base-url-of-deployed/local-api>
```
Comment (saumyakaran) [inline README.md:18]: @akshatbhargava123 We also need to inform the user to restart the server if they make changes to the `.env.local` file

**Comment by saumyakaran** [comment]
@akshatbhargava123 Adding a small request: 

- [ ] Please remove the "learn more" text and link for portfolio value and yield info question marks

I realized that it doesn't make sense to have a "learn more" link for portfolio value and yield... Only a link to how we calculate rewards is sufficient.

**Comment by akshatbhargava123** [comment]
ETA: 24th July, EOD.

File: README.md (hunk 1 above)
Comment (akshatbhargava123) [reply README.md:18]: The streamlined flow already shows that user need to edit `.env.local` before running the development server and I don't feel a need for explicit mention here, do you still think it's worth mentioning?

**Pull request merged**